test2code/build.gradle.kts

```diff
@@ -51,6 +51,7 @@ dependencies {
     implementation(project(":test2code-common"))
     implementation(project(":agent"))
 
+    testImplementation(kotlin("test-junit5"))
     testImplementation("junit:junit:4.13.1")
```

Comment:
Do we need all these dependencies? Is just test-junit5 enough?

Reply:
leave only  test-junit5 and mockito kotlin

Reply:
Do we need  testImplementation("junit:junit:4.13.1")? Just replace kotlin("test-junit5") to kotlin("test-junit") and JUnit-4 will be included automatically.

Reply:
Something goes wrong with comment. Actually testImplementation("junit:junit:4.13.1") is removed. Perfect.